src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -30721,6 +30721,22 @@ unit:MilliBAR
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;
   rdfs:label "Millibar"@en .
 
+unit:MilliBAR_A
+    a qudt:Unit ;
+    dcterms:description "The same as a Millibar, however the Absolute is added when there is a mixture of gauge and absolute measurements to ensure there is a clear distinction"^^qudt:LatexString ;
+    qudt:applicableSystem sou:CGS ;
+    qudt:conversionMultiplier 100.0 ;
+    qudt:conversionMultiplierSN 1.0E2 ;
+    qudt:exactMatch unit:HectoPA ;
```

Comment:
To avoid confusion for those examining unit:HectoPA, I suggest we change this triple to
```
qudt:exactMatch unit:MilliBAR ;
```
Then, in the definition of unit:MilliBAR we must also have
```
qudt:exactMatch unit:MilliBAR_A ;
```
...to satisfy our validation constraint (qudt:ExactMatchGoesBothWaysConstraint).